src/arkit.cs

```diff
@@ -177,9 +210,20 @@ interface ARCamera : NSCopying {
 			get;
 		}
 
+#if !XAMCORE_4_0
+		[Obsolete ("Use 'Project' instead.")]
+		[Wrap ("Project (point, orientation, viewportSize)")]
+		CGPoint GetProjectPoint (Vector3 point, UIInterfaceOrientation orientation, CGSize viewportSize);
```

Comment:
This is a breaking change: it's removing the `virtual` specifier (see the api diff for this PR).